migration_guide.md

```diff
@@ -30,6 +30,8 @@ Note also that this change does *not* affect automatic reconnection attempts in
 ## Receiving data from IoTHub
 Similarly to the above, there is an additional explicit step you must now make when trying to receive data. In addition to setting your handler, you must explicitly start/stop receiving. Note also that the above step of manually connecting must also be done before starting to receive data.
 
+Furthermore, note that the content of the message is now referred to by the 'payload' attribute on the message, rather than the 'data' attribute.
```

Comment:
<disapproving glare> This seems somewhat arbitrary. What's the benefit of this change?

Reply:
Consistency. We call it a 'payload' on other objects. Ryan was on board with this change, but we can add some backwards compatibility support to it as well.